src/components/BlipSelect.vue

```diff
@@ -611,6 +617,9 @@ export default {
       height: 14px;
     }
   }
+  &.readonly{
+    cursor: default
```

Comment:
Isn't better `cursor: not-allowed`? This will give a better experience to user displaying a block icon, saying to user that the click is not enabled.

Reply:
I don't think so, I've considered it, but I think that is not necessary.
For me, the not-allowed seems to mean that the user should perform some action to enable the click, or as the name says, the user is not allowed to click somewhere for some reason. With the default pointer the experience is smoother, since the click is not even a option, the user is not going to be 'not allowed' because the click action doesn't exist.

Reply:
It shouldn't be better to show this situation to a PD?